### ISSUE TITLE
a11y issues with common color combination

### ISSUE THREAD

**Issue by lc512k** [Financial-Times/o-colors#111]
`teal-1` (#27757b) and `pink-tint1` (#f6e9d8) is a very common color combination on Next, as seen in https://www.ft.com/barriers (when signed out)

<img width="297" alt="screen shot 2016-11-07 at 15 47 25" src="https://cloud.githubusercontent.com/assets/3425322/20064413/8b4c5d18-a502-11e6-9fb3-384b92d27da5.png">

It has a contrast ratio of 1.48:1, which is _very_ close to the required 1.5:1 required by the WCAG2.0 but currently breaks `pa11y-ci` tests for Next barriers.

Minimally tweaking either so that the ratio reaches 1.5:1 would allow Next products to continue using this popular color combination while having minimal visual impact

May we suggest #f7e9d9 for `pink-tint1`, which does the trick and looks fairly similar?
<img width="385" alt="screen shot 2016-11-07 at 16 00 42" src="https://cloud.githubusercontent.com/assets/3425322/20064670/628a006e-a503-11e6-9a53-84e22d911553.png">

cc @alicebartlett @rowanmanning

**Comment by alicebartlett** [comment]
Wow these two colours are incredibly close! Let me check with Sue Vago, but this should be an easy yes.

**Comment by alicebartlett** [comment]
Sue's given this a thumbs up, PR is here #112
